SportsHub/SportsHub.DAL/Data/Configurations/UserEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SportsHub.Domain.Models;
+
+namespace SportsHub.DAL.Data.Configurations
+{
+    public class UserEntityTypeConfiguration : IEntityTypeConfiguration<User>
+    {
+        public void Configure(EntityTypeBuilder<User> user)
+        {
+            user.Property(x => x.Email)
+                .IsRequired(true)
+                .HasMaxLength(50)
```

Comment:
move constraints into separate file with constants

Reply:
why have you changed the value of maxLength? i saw another in the database diagram

Reply:
When doing the models, I went with ```50``` instead of ```100``` as ```100``` seemed to many characters for the fields. But this can be updated if we decide to go with ```100```, it just seems too much.

Reply:
I don`t mind changing of limits, but it should be declared and discussed with someone. And there should be at least a message in discord that you have decided to change the scheme

Reply:
Implemented, also I have changed the username limit to 50.